docs/application/dotnet/guides/applications/application_lifecycle.md

```diff
@@ -0,0 +1,186 @@
+# Application lifecycle
+
+The Tizen .NET application model handles application lifecycle and system events. The Tizen .NET application lifecycle is handled by the [Tizen.Applications](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.html) namespace.
+
+It provides methods to manage the main event loop, the application state change events, and basic system events. Applications in the same package follow the same installation lifecycle, which is handled by the application package manager.
+Each application in the package follows its own application lifecycle. Each application in an application package can be identified by its own ID.
+
+<a name="state_change"></a>
+## Tizen .NET application state change methods
+
+The following example shows the basic usage of the lifecycle events provided by the `NUIApplication` class derived from the `CoreApplication` and `Application`. To use the `NUIApplication` class, `Tizen.NUI` namespace have to be included in the project. In the following example, two additional namespaces are used: `Tizen.NUI.BaseComponents` for the `TextLabel` component and `Tizen.Sensor` for the accelerometer access. The application shows reason of overriding the `OnPause` and `OnResume` methods. The app suspends `Accelerometer` update callback to save power in the `Paused` state:
+
+```csharp
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.Sensor;
+```
+
+The main class of the application is defined as `Program`. In this case, the `Program` class derives from the `NUIApplication` object. The `Accelerometer` handle is declared in the main class to use it in all lifecycle callbacks. The `OnCreate()` method uses `Tizen.Log.Info` to print information in `dlogutil` tool. For more information on logging system in the `Tizen`, see [Log Viewer](/application/native/guides/error/system-logs.md#dlogutil). `OnCreate()` method initializes the UI and the `Accelerometer` component:
+
+```csharp
+    class Program : NUIApplication
+    {
+        private Accelerometer SensorAccelerometer = null;
+
+        protected override void OnCreate()
+        {
+            Tizen.Log.Info("LifecycleTest", "OnCreate()");
+            base.OnCreate();
+            InitSensors();
+            InitUI();
+        }
+    }
+```
+
+The following code snippet is responsible for initializing `Accelerometer` sensor. The function setup the event handler for data update and set the update interval for one second. After setup, the `SensorAccelerometer.Start()` to start measurements:
```

Comment:
After setup, the `SensorAccelerometer.Start()` to start measurements:

Question to this sentence. In my thought the `SensorAccelerometer.Start()` is the subject of this sentence. But what is the verb of this sentence?

Reply:
Fixed.